pyproject.toml

```diff
@@ -61,7 +62,7 @@ excludes = [
 ]
 
 [tool.pytest.ini_options]
-addopts = "-rsx --tb=short --loop-scope session"
+addopts = "-rsx --tb=short"
```

Comment:
loop-scope is a new option i've added which parameterizes the scope of the event-loop fixture.  Sharing the same event loop for the entire pytest session allows async fixtures to work at higher scopes.